src/sentry/sentry_metrics/indexer/cloudspanner/cloudspanner.py

```diff
@@ -1,25 +1,65 @@
-from typing import Any, Mapping, Optional, Set
+import logging
+import re
+from datetime import datetime
+from http import HTTPStatus
+from typing import Any, Mapping, Optional, Sequence, Set
 
+import google.api_core.exceptions
 from django.conf import settings
 from google.cloud import spanner
 
 from sentry.sentry_metrics.configuration import UseCaseKey
-from sentry.sentry_metrics.indexer.base import KeyResult, KeyResults, StringIndexer
+from sentry.sentry_metrics.indexer.base import (
+    FetchType,
+    KeyCollection,
+    KeyResult,
+    KeyResults,
+    StringIndexer,
+)
 from sentry.sentry_metrics.indexer.cache import CachingIndexer, StringIndexerCache
-from sentry.sentry_metrics.indexer.id_generator import reverse_bits
+from sentry.sentry_metrics.indexer.cloudspanner.cloudspanner_model import (
+    SpannerIndexerModel,
+    get_column_names,
+)
+from sentry.sentry_metrics.indexer.id_generator import get_id, reverse_bits
+from sentry.sentry_metrics.indexer.ratelimiters import writes_limiter
 from sentry.sentry_metrics.indexer.strings import StaticStringIndexer
+from sentry.utils import metrics
 from sentry.utils.codecs import Codec
 
+logger = logging.getLogger(__name__)
+
 EncodedId = int
 DecodedId = int
 
+_INDEXER_DB_METRIC = "sentry_metrics.indexer.cloudspanner"
```

Comment:
I know that using the postgres metric is wrong, though I need to point out that all the dashboards we have (both in datadog and in grafana for the load test platform) look for the old metric name.

Reply:
datadog: we should be able to mass-replace a keyword by exporting the dashboard as json, doing search and replace there and reimporting it

grafana: no clue

Reply:
I have defaulted back to using postgres for now. The reason I want it to be cloudspanner is when we run this in parallel we need to be able to differentiate between metrics emitted from both the consumers. After the performance runs, we can change this to cloudspanner